iroh-net/src/magicsock/peer_map/endpoint.rs

```diff
@@ -582,7 +511,7 @@ impl Endpoint {
 
     /// Clears all the endpoint's p2p state, reverting it to a DERP-only endpoint.
     #[instrument(skip_all, fields(peer = %self.public_key.fmt_short()))]
-    fn reset(&mut self) {
+    pub fn reset(&mut self) {
```

Comment:
pub(super) please

Reply:
done in 8534b4da7be652ebafdb97266c46ec3c41eb7f03